test_first/test/e2e/todo_test.go

```
@@ -0,0 +1,97 @@
+			if err != nil {
+				t.Fatalf("failed to send request: %v", err)
+			}
+			defer resp.Body.Close()
+
+			if resp.StatusCode != tt.wantStatus {
+				t.Errorf("expected status %d, got %d", tt.wantStatus, resp.StatusCode)
+			}
+
+			if tt.wantError {
+				var responseBody map[string]string
+				json.NewDecoder(resp.Body).Decode(&responseBody)
```

golangci-lint (1.64.8)

Error return value of `(*encoding/json.Decoder).Decode` is not checked

(errcheck)